apps/backend/src/libs/modules/plugins/auth/auth.plugin.ts

```diff
@@ -44,7 +47,7 @@ const auth = (app: FastifyInstance, { whiteRoutes }: PluginOptions): void => {
 			throw new AuthError();
 		}
 
-		request.user = user;
+		request.user = user.toObject() as UserAuthResponseDto;
```

Comment:
it should be a plain user here already
```suggestion
		request.user = user;
```